src/types/rooms.ts

```diff
@@ -50,3 +51,13 @@ export enum RoomTemplate {
 }
 
 export type VenueRoomTemplate = VenueTemplate | RoomTemplate;
+
+type LabelOption = { label: string; value: string };
+
+export const LABEL_VISIBILITY_OPTIONS: LabelOption[] = [
+  { label: "Always", value: RoomVisibility.nameCount },
+  { label: "On Hover", value: RoomVisibility.hover },
+  { label: "Count only", value: RoomVisibility.count },
+  { label: "No Label", value: RoomVisibility.none },
+  { label: "Unclickable", value: RoomVisibility.unclickable },
+];
```

Comment:
Not a type. Best moved into `settings/` directory since now it's easy to make a new file and at the top of `settings/index.ts` add

```ts
export * from "./portal-settings";
```

Reply:
fixed in 2ab4e9997f9341a3a86c13ea0a77a77b55a15131